src/js/base/model.js

```diff
@@ -84,6 +85,8 @@ export default Backbone.Model.extend(extend({
     return isFunction(messages[category]) ? messages[category] : this[messages[category]];
   },
   handleMessage({ category, payload }) {
+    payload.attributes = extend(payload.attributes || {}, { updated_at: dayjs.utc().format() });
```

Comment:
nitpick but I think `extend({}, payload.attributes, updated_at` is probably better since it'll remove the logic branch and it doesn't mutate the `payload.attributes`

Reply:
Agreed, I updated it to that 👍